corehq/apps/users/models.py

```diff
@@ -1722,14 +1722,14 @@ def unset_location(self, fall_back_to_next=False):
         """
         from corehq.apps.fixtures.models import UserFixtureType
         from corehq.apps.locations.models import SQLLocation
-
         old_primary_location_id = self.location_id
-        self.assigned_location_ids.remove(old_primary_location_id)
-        self.get_domain_membership(self.domain).assigned_location_ids.remove(old_primary_location_id)
+        if old_primary_location_id:
+            self.assigned_location_ids.remove(old_primary_location_id)
+            self.get_domain_membership(self.domain).assigned_location_ids.remove(old_primary_location_id)
```

Comment:
@sravfeyn  It is... its github messing you up.. 
https://github.com/dimagi/commcare-hq/blob/deb041314b4762d7ccfcfc55c2833d7728cf8108/corehq/apps/users/models.py#L1726